pkg/arvo/app/hark-store.hoon

```diff
@@ -653,6 +655,13 @@
     =>  (emit autoseen-timer)
     poke-core(current-timebox now.bowl)
   ::
+  ++  read-all
+    =:  unreads-count  (~(run by unreads-count) _0)
+        unreads-each    (~(run by unreads-each) _~)      
+        notifications  (~(run by notifications) _~)
```

Comment:
this will have to rebuild the cache, if you're going to clear the notifications as well. Just change `seen` to 
```
=>  rebuild-cache
seen
```